app-backend/migrations/src_migrations/main/scala/76.scala

```diff
@@ -0,0 +1,16 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M76 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(76)(List(
+    sqlu"""
+CREATE TABLE layer_attributes (
```

Comment:
It we tweak this to be `CREATE TABLE IF NOT EXISTS ...`, then we can keep the table that was created in staging (out-of-band) in tact.